### ISSUE TITLE
Docker File for v1.8

### ISSUE THREAD

**Issue by Pratyush1991** [remicres/otbtf#18]
Hello, 
I would like to know if otbtf1.8 is available on Docker Hub.? Or should I use the same docker file as previous (otbtf1.7) to build the latest one for GPU.?

**Comment by remicres** [comment]
The docker image for 1.8 is not ready for now, but yes you can update the repo (`/work/otb/otb/Modules/Remote/otbtf`) and recompile from OTB build dir (`/work/otb/build/OTB/build`) it should be straightforward

**Comment by Pratyush1991** [comment]
Thank you. It did work.

I find that after each commit for docker image a new image is created even I tried to delete the old docker images (parent image) but it says error that the image has an independent child and cannot be deleted. Even I searched online for solution I couldn't find any and I found that it say each new docker image is like a layer which are shared from previous so parent image cannot be deleted. As each new docker image size is around 23 GB (for my case) so I'm a bit confused that if it just adds to the parent image or a new docker image is created every time which occupies 23 GB.
